app.js

```diff
@@ -1,17 +1,186 @@
 /* Imports */
-// import { getRandomItem } from './utils.js';
+import { getRandomItem } from './utils.js';
+
+// grab DOM elements
+const guess1 = document.getElementById('guess-1');
+const guess2 = document.getElementById('guess-2');
+const guess3 = document.getElementById('guess-3');
+
+const shell1 = document.getElementById('shell-1');
+const shell2 = document.getElementById('shell-2');
+const shell3 = document.getElementById('shell-3');
+
+const pearl1 = document.getElementById('pearl-1');
+const pearl2 = document.getElementById('pearl-2');
+const pearl3 = document.getElementById('pearl-3');
+
+const display1 = document.getElementById('display-1');
+const display2 = document.getElementById('display-2');
+const display3 = document.getElementById('display-3');
+
+const guesses = document.getElementById('guesses');
+// const results = document.getElementById('results');
+const playAgainButton = document.getElementById('play-again-button');
+
+const winsDisplay = document.getElementById('wins-display');
+const lossesDisplay = document.getElementById('losses-display');
+const totalDisplay = document.getElementById('total-display');
 
 /* State */
+let wins = 0;
+let losses = 0;
+
+let gameState = 'guess'; // 'guess' or 'results'
+let pick = ''; // 'guess1', 'guess2', or 'guess3'
+let pearl = ''; // 'pearl1', 'pearl2', or 'pearl3'
 
 /* Actions */
-function loadPage() {}
+function loadPage() {
+    if (gameState === 'guess') {
+        displayShells();
+    } else if (gameState === 'results') {
+        displayResults();
+    }
+}
+
+function displayScoreboard() {
+    winsDisplay.textContent = wins;
+    lossesDisplay.textContent = losses;
+    totalDisplay.textContent = wins + losses;
+}
+
+const locations = [pearl1, pearl2, pearl3];
 
-/* Components */
+function findPearl() {
+    gameState = 'results';
```

Comment:
i think this line might be making it run ```displayResults()``` again? something is making it call twice